Tools/Create-Azure-Sentinel-Solution/createSolution.ps1

```diff
@@ -903,22 +1146,30 @@ foreach ($inputFile in $(Get-ChildItem $path)) {
                                 }
                                 $baseCreateUiDefinition.parameters.steps += $huntingQueryBaseStep
                             }
+
                             $huntingQueryObj = [PSCustomObject] @{
-                                type       = "savedSearches";
+                                type       = $contentToImport.TemplateSpec ? "Microsoft.OperationalInsights/savedSearches" : "savedSearches";
                                 apiVersion = "2020-08-01";
-                                name       = "$solutionName Hunting Query $huntingQueryCounter";
-                                dependsOn  = @(
-                                    "[variables('workspace-dependency')]"
-                                );
+                                name       = $contentToImport.TemplateSpec ? "$($solutionName.Replace(' ', '_'))_Hunting_Query_$huntingQueryCounter" : "$solutionName Hunting Query $huntingQueryCounter";
```

Comment:
Can you check the else part in the name section?

Reply:
it is same as earlier, shouldn't create problem